.reuse/dep5

```diff
@@ -0,0 +1,660 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Gardener
+Upstream-Contact: The Gardener project <gardener@googlegroups.com>
+Source: https://github.com/gardener/gardener
+Comment: The source code of this component was seeded based on a copy of the following files from https://github.com/kubernetes:
+  .
+  Sample APIServer
+  http://git.k8s.io/sample-apiserver
+  Copyright 2017 The Kubernetes Authors
+  Apache 2 license (https://git.k8s.io/sample-apiserver/LICENSE)
+  .
+  Release: 1.9
+  Commit-ID: 7c9967f6de296b968505885781e9ed9fc65156c3
+  Commit-Message: Merge remote-tracking branch 'origin/master' into release-1.9
+  .
+  Kubernetes
+  https://git.k8s.io/kubernetes
+  Copyright 2017 The Kubernetes Authors
+  Apache 2 license (https://git.k8s.io/kubernetes/LICENSE)
+  .
+  The following charts of the `charts` directory were seeded based on a copy of the following origins:
+  * https://git.k8s.io/charts/stable/kube-state-metrics
+  * https://git.k8s.io/charts/stable/nginx-ingress
+  * https://git.k8s.io/charts/stable/prometheus
+  * https://git.k8s.io/charts/stable/kubernetes-dashboard
+  * https://github.com/grafana/loki/blob/master/production/helm/loki/templates/statefulset.yaml
+  * https://github.com/grafana/loki/blob/master/production/helm/loki/templates/service.yaml
+  .
+  Container Images:
+  The list of container images and versions deployed by the Gardener can be found [here](charts/images.yaml).
+
+# --------------------------------------------------
+# source code
+
+Files: **
+Copyright: 2017-2020 SAP SE or an SAP affiliate company and Gardener contributors
+License: Apache-2.0
+
+# --------------------------------------------------
+# documentation
+
+Files:
+  docs/**
+  ADOPTERS.md
+  CONTRIBUTING.md
+  README.md
+  SECURITY.md
+Copyright: 2017-2020 SAP SE or an SAP affiliate company and Gardener contributors
+License: CC-BY-4.0
+
+# --------------------------------------------------
+# dependencies
+
+Files: vendor/github.com/ahmetb/gen-crd-api-reference-docs/**
+Copyright: 2018-2020 Ahmet Alp Balkan
+License: Apache-2.0
```

Comment:
these are created semi-automated at the moment and I filed
https://github.wdf.sap.corp/OSPO/OSPO-request/issues/472
for OSPO to get help from them since it's too much work to do this manually